en/developer_guide/block_display.rst

```diff
@@ -0,0 +1,105 @@
+Block display
+=============
+
+Display Independence
+--------------------
+
+In the Front Office, some blocks could require some specific stylesheets and javascript files.
+To allow any block to add some specific files, the main node layout loads the
+`requireJS`_ library which is a JavaScript file and module loader.
+Each block is allowed to call the ``require()`` function of the ``requireJS`` library.
+To allow stylesheets dynamic load, Open Orchestra offers an AMD module named ``openOrchestraCSS``
+which is meant to be use with ``requireJS``.
+In a case in which the same block is displayed several times, stylesheets and javascript files
+corresponding will be loaded only once.
+
+Block loading file
+-------------------
+
+For each block there is a ``blockLoader.js`` file using the ``requireJS`` library to load the
+requirements needed to initialize a block.
+The  required elements are the module used to load stylesheets: ``openOrchestraCss``,
+specific javascript files in block and javascript files in common libraries.
+
+After loading the elements, it does the initialization. During this step, javascript is executed
```

Comment:
"the JavaScript code is run" whould be better
But, it should be precised what code is run, is it only the code specific to a block ?